app/assets/javascripts/discourse/components/ignored-user-list-item.js.es6

```diff
@@ -0,0 +1,9 @@
+export default Ember.Component.extend({
+  tagName: "div",
+  items: null,
+  actions: {
+    removeItem(item) {
+      this.send("removeItem", item);
```

Comment:
Just realised, we can't actually do that, we are communicating back to the parent component. We should keep it as it originally was.